csrankings-h.csv

```diff
@@ -266,7 +266,7 @@ Hao Chen 0003,Univ. of California - Davis,http://web.cs.ucdavis.edu/~hchen,1Aa3q
 Hao Chen 0011,HKUST,https://cse.hkust.edu.hk/~jhc,Z_t5DjwAAAAJ
 Hao Dong 0003,Peking University,https://zsdonghao.github.io,xLFL4sMAAAAJ
 Hao Li 0011,Xi'an Jiaotong University,https://aquatoney.github.io,-rpSVLEAAAAJ
-Hao Li 0015,University of Southern California,http://hao-li.com,NFeigSoAAAAJ
+Hao Li 0015,MBZUAI,https://mbzuai.ac.ae/study/faculty/hao-li,NFeigSoAAAAJ

```

Comment:
CEO of a startup (Pinscreen); seems incompatible with a full-time faculty position. Please clarify.

Reply:
Please see official confirmation letter from MBZUAI HR Director as an email attachment

Reply:
Please see official confirmation in letter from MBZUAI HR Director as an email attachment